Marlin/Marlin_main.cpp

```diff
@@ -347,20 +347,31 @@ bool target_direction;
 #endif
 
 #ifdef DELTA
+
+  #define TOWER_1 X_AXIS
+  #define TOWER_2 Y_AXIS
+  #define TOWER_3 Z_AXIS
```

Comment:
Id vote for "#define TOWER_A X_AXIS" ect. if we are making A,B,C the default throughout marlin. 

I do admit, (putting the HAL argument aside) using the "axis" term when talking about the towers on delta printers gets a little confusing :/